src/main/java/com/google/sps/data/Route.java

```diff
@@ -26,6 +26,7 @@
   private long startMinute;
   private List<Marker> routeMarkers;
   private List<Long> editorsArray;
+  private String status;
```

Comment:
If a field will only contain a fixed set of values (e.g. new, copy, edit, delete) it is better to use an enum. Enums ensure that you can only use the possible values and not have unexpected ones.Typos will be caught at compile time rather than runtime. The earlier you can catch potential errors, the better.

Reply:
Done